orchagent/portsorch.cpp

```diff
@@ -7416,7 +7494,20 @@ bool PortsOrch::setPortSerdesAttribute(sai_object_id_t port_id, sai_object_id_t
     port_serdes_attr.id = SAI_PORT_SERDES_ATTR_PORT_ID;
     port_serdes_attr.value.oid = port_id;
     attr_list.emplace_back(port_serdes_attr);
-    SWSS_LOG_INFO("Creating serdes for port 0x%" PRIx64, port_id);
+    SWSS_LOG_DEBUG("Creating serdes for port 0x%" PRIx64 " with the following SI parameters:", port_id);
```

Comment:
@tshalvi can we rely upon `/var/log/swss.rec ` for debug log?

Reply:
Yea, you're right. I'm removing this log and the following loop.